test/ered_SUITE.erl

```diff
@@ -70,8 +69,15 @@ wait_for_consistent_cluster() ->
                                 ered_client:stop(Pid),
                                 SlotMap
                         end(P) || P <- ?PORTS],
+            %% Get the amount of nodes in the first reply. The reply format has improved between
+            %% Redis 6 and 7, but to handle Redis 6 we count each expected port in the reply.
+            NoOfNodes = length([X || X <- [lists:member(integer_to_binary(P),
+                                                        binary:split(list_to_binary(hd(AllNodes)), <<"\n">>, [global]))
+                                           || P <- ?PORTS],
+                                     X =:= true]),
+
```

Comment:
Is there a simpler way? (probably..)
Redis7 divides nodes with `\n\n` and then a `\n` between each info element for the node,
Redis 6 is just single newlines.

Reply:
What? THis is CLUSTER SLOTS, it should be a nested list without any newlines.

I see now that an ered_client connection is established, then it is not used but instead redis-cli is used via os:cmd to call CLUSTER SLOTS. Then the ered_client connection is closed. Obviously a mistake but if all we were interested in was that all nodes return identical information, this seemed to have worked..

Reply:
Yikes.. I must have got temporary code blindness.. missed that. Thanks